content/en/news/releases/1.14.x/announcing-1.14.5/index.md

```diff
@@ -0,0 +1,31 @@
+---
+title: Announcing Istio 1.14.5
+linktitle: 1.14.5
+subtitle: Patch Release
+description: Istio 1.14.5 patch release.
+publishdate: 2022-10-07
+release: 1.14.5
+---
+
+This release includes security fixes in go 1.18.7 (released 2022-10-04) for the `archive/tar`, `net/http/httputil`, and `regexp` packages.
+This release also includes fixes to improve robustness.
+This release note describes what’s different between Istio 1.14.4 and Istio 1.14.5.
+
+{{< relnote >}}
+
+## Changes
+
+- **Fixed** `kube-inject` crashes when the pod annotation `proxy.istio.io/config` is set.
+
+- **Fixed** an issue where user can not delete `iop` resource with revision if istiod is not running.  ([Issue #40796](https://github.com/istio/istio/issues/40796))
+
+- **Fixed** an issue that default `idleTimeout` for passthrough cluster has been changed to `0s` since 1.14.0 and timeout is disabled. Previous behavior is using Envoy's default value.
```

Comment:
Comment added ini other RN: I'm not really sure what this fix is about. We mention pre 1.14.0 we used Envoy's default value. In 1.14.0 the default is 0 and the timeout is disabled. Now we fixed it, but what did we fix it to?

Also changes in wording.

Reply:
@fatedier Can you take a look at the comment above?

Reply:
The 1.15.2 RN PR has a wording change and this was my latest suggestion there:

```suggestion
- **Fixed** an issue that the default `idleTimeout` for the passthrough cluster was changed to `0s` in 1.14.0, disabling the timeout. Restored the previous behavior to using Envoy's default value of 1 hour. ([Issue #41114](https://github.com/istio/istio/issues/41114))
```

Reply:
Thanks, Eric. Your suggestion is merged.